components/spin/index.tsx

```diff
@@ -146,8 +171,8 @@ const Spin: SpinType = (props) => {
 
   const mergedIndicator = indicator ?? contextIndicator ?? defaultIndicator;
 
-  const rootStyle: React.CSSProperties = { ...contextStyles.root, ...styles?.root };
-  const wrapStyle: React.CSSProperties = { ...contextStyles.wrapper, ...styles?.wrapper };
+  const rootStyle: React.CSSProperties = { ...mergedStyles.root };
+  const wrapStyle: React.CSSProperties = { ...mergedStyles.wrapper };
```

Comment:
```suggestion
```

Reply:
现在没有合并的话，就可以考虑直接删掉这边然后在下面用

```suggestion
  style={mergedStyles.wrapper}
```